### ISSUE TITLE
Colored tags

### ISSUE THREAD

**Issue by giann** [GothenburgBitFactory/taskwarrior#2567]
Would it be possible to be able to chose a color for each tag and see them printed in that color in the Tag column?

**Comment by tbabej** [comment]
HI @giann,

there is a `color.tag.<name>` configuration setting that you can use, however, that will not assign the color only to the tag, but to the entire task line if the task contains that tag.

**Comment by giann** [comment]
Thanks. I would have loved a color for the tag  name alone but this sort of gets me what I want.

**Comment by tbabej** [comment]
Yeah I do think that's an interesting proposition indeed, but would have to be a part of bigger feature :slightly_smiling_face: I'm gonna keep this feature request around and hopefully we can get to it at some point in the future.
